core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -1407,18 +1424,21 @@ private void buildMatViewGraph() {
                         }
 
                         final MatViewRefreshState state = matViewGraph.addView(matViewDefinition);
-                        final boolean isMatViewStateExists = TableUtils.isMatViewStateFileExists(configuration, path, tableToken.getDirName());
-                        path.trimTo(pathLen).concat(tableToken.getDirName()).concat(MatViewRefreshState.MAT_VIEW_STATE_FILE_NAME);
-                        if (isMatViewStateExists) {
-                            reader.of(path.$());
-                            MatViewRefreshState.readFrom(reader, state);
-                        } else {
-                            blockFileWriter.of(path.$());
-                            MatViewRefreshState.append(state, blockFileWriter);
-                        }
+                        // can be null if the graph has no-op implementation
```

Comment:
can we elaborate what is no-op implementation of the view?

Reply:
Not a view, but the graph. Improved the comment in 75f0e27fa31d0eecdb4df94cff44384ccf3b1e8f